Dockerfile.forms

```diff
@@ -1,27 +1,35 @@
 FROM alpine:edge
+
+USER root
```

Comment:
do we need to keep this in?

Reply:
I was having permission problems during the container build and add the user as root made the problem pass. 😢